src/Chainweb/Rosetta/RestAPI.hs

```diff
@@ -118,9 +137,23 @@ validateNetwork v (NetworkId bc n msni) = do
     SubNetworkId cid _ <- msni ?? RosettaChainUnspecified
     readChainIdText v cid ?? RosettaInvalidChain
 
+
 -- | Guarantees that the `ChainId` given actually belongs to this
 -- `ChainwebVersion`.
 readChainIdText :: ChainwebVersion -> T.Text -> Maybe ChainId
 readChainIdText v c = do
   cid <- readMaybe @Word (T.unpack c)
   mkChainId v cid
+
+
+kdaToRosettaAmount :: Decimal -> Amount
```

Comment:
Suggest a quick unit test for just this function

Reply:
I merged this PR into the #1050 and added a unit test for `kdaToRosettaAmount` there